src/Public/Send-OSNotification.ps1

```diff
@@ -1,24 +1,33 @@
 <#
 .SYNOPSIS
-Short description
+Displays toast notifications on Linux, macOS, and Windows.
 
 .DESCRIPTION
-Long description
+The Send-OSNotification function displays notification on Linux, macOS, and Windows.
+
+Each operating system uses different methods for generating and displaying notifications, but this function abstracts 
+each of them for a consistent experience regardless of the environment in which it is used.
 
 .PARAMETER Body
-Parameter description
+The main text of your notification. You must specify a body string.
 
 .PARAMETER Title
-Parameter description
+The title of your notification. Generally this is rendered bold at the top of the notification.
+
+Defaults to "PowerShell Notification".
 
 .PARAMETER Icon
-Parameter description
+If specified, the specified picture is included on the notification.
+
+There is a default Icon included on Windows only.
```

Comment:
PSNotifySend also sets a default PowerShell logo icon. Maybe include Linux here.

Reply:
Should we consider putting this module's default through to both linux and mac? (e.g. put the relative path into the public function like the default title)

Reply:
Right now, there's some weirdness on Linux on how the icon is set. To make a long story short, PSNotifySend has some work that needs to be done to support that.

And I don't know if I like that idea on macOS because that means we're only using the `Alerter Notification` cmdlet and that one blocks execution until the notification disappears...

So maybe for now, let's just say Windows is the only one with a default icon since that's the only one with our icon as the default icon. I'll fix PSNotifySend to handle Icons better... As for macOS... I think we should just try to use the Alerter cmdlet as little as possible. That blocking is kinda a pain point... At least in my opinion.

Reply:
Tl;Dr keep what you have and merge this PR 😊